PROTO_tests/tests/alignment_test.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import arkouda as ak
```

Comment:
this is kinda on me for not mentioning it last time, we also try and run `isort` to manage our imports. This isn't a huge deal especially for tests, but it's def something to get in the habit of when u modify actual arkouda files

Reply:
Updated